syntax = "proto3";

package micromissiles;

import "configs/proto/agent.proto";

// Simulator configuration.
message SimulatorConfig {
  // End time in seconds.
  float end_time = 1;

  // Time scale.
  float time_scale = 2;

  // Interceptor swarm configurations.
  repeated AgentSwarmConfig interceptor_swarm_configs = 3;

  // Threat swarm configurations.
  repeated AgentSwarmConfig threat_swarm_configs = 4;
}
